### PR TITLE
add iot links section

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

**[Smart displays](https://ubuntu.com/internet-of-things/smart-displays)**
- Added cards linking to other IoT pages - [copy doc](https://docs.google.com/document/d/1fFvBl-ZZvKDFV6OMv7O4kudQKtyxfvSU60rfVdKluAQ/edit#heading=h.qkat7vybg1m)

**[Gateways](https://ubuntu.com/internet-of-things/gateways)**
- Added cards linking to other IoT pages - [copy doc](https://docs.google.com/document/d/18ffTcNolrtJS9AvjMfa8KW94uLfC0KzmHtTe-jMYmlo/edit#heading=h.nby4fxmsrrg3)

**[Automotive](https://ubuntu.com/automotive)**
- Added cards linking to other IoT pages - [copy doc](https://docs.google.com/document/d/1PVliB92h4ZqexaZV9F5MIcX7aRG6yBAGH1hG7x8UZTw/edit#heading=h.7zytw966kvza)

**[Networking](https://ubuntu.com/internet-of-things/networking)**
- Added cards linking to other IoT pages - [copy doc](https://docs.google.com/document/d/1EWlkTkGvul-ek4P1cR9pyNvyJsh5WtpOkChvk0igzVQ/edit#)

**[Smart city](https://ubuntu.com/internet-of-things/smart-city)**
- Added cards linking to other IoT pages - [copy doc](https://docs.google.com/document/d/1Zh-IkLDnaqmPr4TTeUZ3mtyY2_OGW4iElfntYWZyveY/edit#)

## QA

Visit:
- https://ubuntu-com-11758.demos.haus/internet-of-things/smart-displays
- https://ubuntu-com-11758.demos.haus/internet-of-things/gateways
- https://ubuntu-com-11758.demos.haus/automotive
- https://ubuntu-com-11758.demos.haus/internet-of-things/networking
- https://ubuntu-com-11758.demos.haus/internet-of-things/smart-city

See that the new section has been added to the bottom of each page

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5236

## Screenshots
image

### CODE REVIEW

**Comment by davegoddard42** [comment]
LGTM @sowasred2012

File: templates/internet-of-things/shared/_links.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      <a href="/automotive">Learn more&nbsp;&rsaquo;</a>
+    </div>
+    {% endif %}
+
+    {% if current != "networking" %}
+    <div class="col-3 p-divider__block">
+      <h2 class="p-heading--3">Networking</h2>
+
+      <p>Build secure SmartNICs and operate them at scale on Ubuntu.</p>
+      <a href="/networking">Learn more&nbsp;&rsaquo;</a>
```
Comment (steverydz) [inline templates/internet-of-things/shared/_links.html:36]: This `href` should be `/internet-of-things/networking`

**Review by steverydz — APPROVED** [review]
Looks good - just the networking link has the wrong URL

File: templates/internet-of-things/shared/_links.html (hunk 1 above)
Comment (sowasred2012) [reply templates/internet-of-things/shared/_links.html:36]: :+1:

**Pull request merged**